product_docs/docs/epas/11/epas_guide/01_introduction/index.mdx

```diff
@@ -18,6 +18,7 @@ The following features have been changed in EDB Postgres Advanced Server 10 to c
 -   Advanced Server now supports *data redaction*, which is a technique for limiting the exposure of sensitive data to certain users. Data redaction results in the alteration of the displayed data to such users. This is accomplished by defining redaction policies for tables with sensitive data. For more information, see *EDB Postgres Advanced Server Security Features Guide*.
 -   Advanced Server now supports *EDB wait states*, which is a background worker that probes each running session at a regular interval collecting information such as the database connection, the logged in user, the running query, and the wait events. For more information, see [Performance Analysis and Tuning](../13_performance_analysis_and_tuning/#edb_wait_states).
 -   Advanced Server now permits an infinite number of tries for custom plan generation by setting the `edb_custom_plan_tries` parameter to `-1`. For more information, see [Query Tuning / Planner Method Configuration](../03_database_administration/01_configuration_parameters/03_configuration_parameters_by_functionality/05_query_tuning_planner_method_configuration/#edb_custom_plan_tries).
+-   Advanced Server no longer supports `edb-pgagent`. The `edb-pgagent` is deprecated in EDB Postgres Advanced Server 11 and now uses the community version of pgAgent.
```

Comment:
```suggestion
-    EDB Postgres Advanced Server 11 now uses the community version of pgAgent and no longer supports `edb-pgagent`.
```

Reply:
To keep it consistent, made minor tweak as:
Advanced Server 11 now uses the community version of pgAgent and no longer supports `edb-pgagent`.
Pls merge with develop. Thanks!